turbinia/jobs/__init__.py

```diff
@@ -33,11 +44,28 @@ def get_jobs():
   from turbinia.jobs.sshd import SSHDAnalysisJob
   from turbinia.jobs.tomcat import TomcatExtractionJob
   from turbinia.jobs.tomcat import TomcatAnalysisJob
+
+  jobs_blacklist = jobs_blacklist if jobs_blacklist else []
+  jobs_whitelist = jobs_whitelist if jobs_whitelist else []
+
   # TODO(aarontp): Dynamically look up job objects and make enabling/disabling
   # configurable through config and/or recipes.
-  return [
-    StatJob(), PlasoJob(), PsortJob(), StringsJob(), GrepJob(),
-    SSHDExtractionJob(), SSHDAnalysisJob(), TomcatExtractionJob(), TomcatAnalysisJob()]
+  jobs = jobs_list if jobs_list else [
+      StatJob(), PlasoJob(), PsortJob(), StringsJob(), GrepJob(),
+      SSHDExtractionJob(), SSHDAnalysisJob(), TomcatExtractionJob(),
+      TomcatAnalysisJob()]
+
+  if jobs_whitelist and jobs_blacklist:
+    log.info(
```

Comment:
Just .info? I would have thought Warn, at least. I also would prefer raising here, rather than trying to continue - what do you think?

Reply:
+1 on raising and let the user know.

Reply:
Yeah, I think raising is a better idea as well.  Done.